tools/validators/instance_validator/tests/generate_universe_test.py

```diff
@@ -20,6 +20,11 @@
 
 from validate import generate_universe
 from absl.testing import absltest
+from os import path
+
+_DEFAULT_ONTOLOGY_LOCATION = path.join('..', '..', '..',
+                                       'ontology', 'yaml', 'resources')
+_BAD_MODIFIED_ONTOLOGY = path.join('.', 'fake_bad_modified_ontology')
```

Comment:
where is it loaded from ?

Reply:
Loaded from test files which were copied and modified from existing ontology